testbench/test-create/vehicle-app-template/package-index.json

```diff
@@ -9,7 +9,7 @@
                 "args": [
                     {
                         "id": "src",
-                        "description": "URI or path to VSS json (Leave empty for default: v3.0)",
+                        "description": "URI or path to VSS json",
```

Comment:
Why this change? Looked good in the prompt imo.

Reply:
because due to the default value the prompt looks like this:
`Config 'src' for interface 'vehicle-signal-interface': URI or path to VSS json (https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json)`

Reply:
OK👍